frontend/src/components/project-menu/ProjectMenuCard.tsx

```diff
@@ -37,7 +37,8 @@ export function ProjectMenuCard({
 
   const handlePushToGitHub = () => {
     const rawEvent = {
-      content: "Please commit and push these changes to the repository.",
+      content:
+        "Please create a new branch and commit the changes. Then push them to the remote repository, and open up a pull request using the GitHub API and the token in the GH_TOKEN environment variable",
```

Comment:
Isn't this GITHUB_TOKEN?

Reply:
In the new UI, if we have the users GitHub token (e.g., after they sign in), we send a command event that exports the token in the shells environment so the agent can use it

https://github.com/All-Hands-AI/OpenHands/blob/e4590699417506b84c29970895a9be48a9f1fec8/frontend/src/routes/app.tsx#L125

Reply:
Sorry, I'm still confused. The var exists in a .ts file, in .md files including knowledge of a micro agent, and other places, and its name is GITHUB_TOKEN. I don't see how an agent will know about a GH_TOKEN instead. Why aren't we using GITHUB_TOKEN here too?

e.g. https://github.com/search?q=repo%3AAll-Hands-AI%2FOpenHands%20GITHUB_TOKEN&type=code

Reply:
Good catch, I see we even reference it as `GITHUB_TOKEN` in the user suggestions

https://github.com/All-Hands-AI/OpenHands/blob/59fbc11afeb4a7d9dd3ae4dd2a7e05b34421dd8b/frontend/src/utils/suggestions/non-repo-suggestions.ts#L9-L10

Reply:
Yes, exactly. Thanks for the fix!